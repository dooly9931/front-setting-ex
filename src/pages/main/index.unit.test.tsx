import React from "react";
import { shallow } from "enzyme";
import Main from "./index";

describe("Home", () => {
  it("should render home page correctly", () => {
    // const component = shallow(<Main />);
    // const containerWrapper = component.find("#main-page");
    // expect(containerWrapper.length).toBe(1);
    // expect(component).toMatchSnapshot();
  });
});
