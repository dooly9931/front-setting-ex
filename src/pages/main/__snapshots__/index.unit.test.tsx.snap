// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render home page correctly 1`] = `
<div
  id="main-page"
>
  <div>
    <div
      className="title-25 primary-orange"
    >
      Main page
    </div>
    <div
      className="subtitle-16 black"
    >
      Page Introduction: this is ted's main page
    </div>
    <button
      id="wallet-connect"
      type="button"
    >
      Connect Wallet
    </button>
  </div>
  <div>
    <div
      className="subtitle-16"
    >
      Web3 random number: 
    </div>
    <div
      className="subtitle-16"
    >
      URL argument: ""
    </div>
    <div
      className="subtitle-16"
    >
      URL argument - backend response: ""
    </div>
  </div>
</div>
`;
